### ISSUE TITLE
Check folder structure when publishing binaries

### ISSUE THREAD

**Issue by Wambere** [onaio/fhir-tooling#195]
When publishing binary resources, we extract the name of the file from the composition resource and try to find the file in the project folder. This search assumes that the project is already organized in the recommended structure.

This issue is to implement a check to confirm that the folder is actually in the expected structure before trying to find the file.
If the project is not set up in the recommended structure, we can attempt to find the file in the main project folder, and if that does not work, simply show an error and skip the file

**Comment by pld** [comment]
Makes sense, I'd argue beyond assisting with uploading, make it easier to reuse and find information in project folders we want their structure to be consistent and a method to enforce that, which this is

**Comment by pld** [comment]
Noted by @ndegwamartin this is also a prereq to use translation tooling

**Comment by ndegwamartin** [comment]
[Proposed project app config and fhir resource content structure](https://github.com/opensrp/fhircore/discussions/2787)

**Comment by pld** [comment]
Sounds like we're not using that structure, only requirement to verify is

```
/configs
/fhir_content
```

I don't know if translations require an structure beyond that, I guess I have to assume the main IG/composition is in /configs
